sigstore/_internal/rekor/client.py

```diff
@@ -82,114 +82,22 @@ def to_entry(self) -> RekorEntry:
             signed_entry_timestamp=self.signed_entry_timestamp,
         )
 
-
-@dataclass(frozen=True)
```

Comment:
These structures are included in the `SigningResult` (and are used in the verify module) so I moved them to a top-level `rekor` module.

Reply:
Makes sense!